### PR TITLE
Several ActionCable doc fixes

### PR DESCRIPTION
**Pull request by qrush**
Correcting indentation, grammar, and a few other nits. :lock_with_ink_pen:

### CODE REVIEW

File: actioncable/lib/action_cable/server/broadcasting.rb [hunk 1]
```diff
@@ -4,19 +4,19 @@ module Server
...
+    #       stream_from "web_notifications_#{current_user.id}"
+    #     end
+    #   end
     #
-    #    # Somewhere in your app this is called, perhaps from a NewCommentJob
-    #    ActionCable.server.broadcast \
-    #      "web_notifications_1", { title: 'New things!', body: 'All shit fit for print' }
+    #   # Somewhere in your app this is called, perhaps from a NewCommentJob
+    #   ActionCable.server.broadcast \
+    #     "web_notifications_1", { title: 'New things!', body: 'All that's fit for print' }
```
Comment (vipulnsward) [inline actioncable/lib/action_cable/server/broadcasting.rb]: :8ball:

**Comment by vipulnsward** [comment]
@qrush needs a squash of commits + [ci skip]  in the commit message.

File: actioncable/lib/action_cable/channel/base.rb [hunk 2]
```diff
@@ -32,7 +32,7 @@ module Channel
     #
     # == Action processing
     #
-    # Unlike Action Controllers, channels do not follow a REST constraint form for its actions. It's a remote-procedure call model. You can
+    # Unlike ActionControllers, channels do not follow a REST constraint form for their actions. It's a remote-procedure call model. You can
```
Comment (matthewd) [inline actioncable/lib/action_cable/channel/base.rb]: I don't think we've ever used ActionControllers (with or without the space) to mean "instances of AC::Base". It feels awkward to me, but I don't have a concrete suggestion for an alternative. :confused:

**Comment by qrush** [comment]
Rebased. :hocho:

File: actioncable/lib/action_cable/channel/base.rb (hunk 2 above)
Comment (qrush) [inline actioncable/lib/action_cable/channel/base.rb]: Good point, felt weird to me too. Reworked to say "subclasses of".

File: actioncable/lib/action_cable/channel/base.rb [hunk 3]
```diff
@@ -32,8 +32,11 @@ module Channel
     #
     # == Action processing
     #
-    # Unlike Action Controllers, channels do not follow a REST constraint form for its actions. It's a remote-procedure call model. You can
-    # declare any public method on the channel (optionally taking a data argument), and this method is automatically exposed as callable to the client.
+    # Unlike subclasses of ActionController::Base, channels do not follow a REST
+    # constraint form for their actions. It's a remote-procedure call model. You
```
Comment (maclover7) [inline actioncable/lib/action_cable/channel/base.rb]: Instead, ActionCable operates through a remote-procedure call model.

File: actioncable/lib/action_cable/channel/base.rb [hunk 4]
```diff
@@ -60,18 +63,22 @@ module Channel
     #       end
     #   end
     #
-    # In this example, subscribed/unsubscribed are not callable methods, as they were already declared in ActionCable::Channel::Base, but #appear/away
-    # are. #generate_connection_token is also not callable as its a private method. You'll see that appear accepts a data parameter, which it then
-    # uses as part of its model call. #away does not, it's simply a trigger action.
+    # In this example, subscribed/unsubscribed are not callable methods, as they
+    # were already declared in ActionCable::Channel::Base, but <tt>#appear</tt>
+    # and <tt>#away</tt> are. <tt>#generate_connection_token</tt> is also not
+    # callable as its a private method. You'll see that appear accepts a data
```
Comment (maclover7) [inline actioncable/lib/action_cable/channel/base.rb]: its --> it's

File: actioncable/lib/action_cable/channel/base.rb [hunk 5]
```diff
@@ -60,18 +63,22 @@ module Channel
...
+    # parameter, which it then uses as part of its model call. <tt>#away</tt>
+    # does not, since it's simply a trigger action.
     #
-    # Also note that in this example, current_user is available because it was marked as an identifying attribute on the connection.
-    # All such identifiers will automatically create a delegation method of the same name on the channel instance.
+    # Also note that in this example, <tt>current_user</tt> is available because
```
Comment (maclover7) [inline actioncable/lib/action_cable/channel/base.rb]: add a comma after available

**Comment by maclover7** [comment]
Few small grammar notes :smile:

File: actioncable/lib/action_cable/channel/base.rb (hunk 5 above)
Comment (matthewd) [inline actioncable/lib/action_cable/channel/base.rb]: :-1:

Comment (qrush) [inline actioncable/lib/action_cable/channel/base.rb]: Yep, no comma here.

**Comment by qrush** [comment]
Rebased again + a few more nits from @maclover7. I think this is ready to merge.

File: actioncable/lib/action_cable/server/broadcasting.rb (hunk 1 above)
Comment (matthewd) [inline actioncable/lib/action_cable/server/broadcasting.rb]: Sorry, didn't spot this before: unescaped quote

Comment (qrush) [inline actioncable/lib/action_cable/server/broadcasting.rb]: Nice catch, fixed + force pushed (again).

**Comment by rafaelfranca** [comment]
Thanks!!!!

**Pull request merged**